db/db-up.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+if ! test -d "$STATEDIR"
+then
+    echo "$STATEDIR folder not found -- are you running this from the repository root?"
+    exit 1
+fi
+
+echo "PGDATABASE $PGDATABASE"
+echo "PGDATA $PGDATA"
+echo "PGHOST $PGHOST"
+
+trap 'pg_ctl -D $PGDATA stop' EXIT
+
+if ! test -d "$PGDATA"
+then
+    echo "PGDATA directory $PGDATA does not exist -- running setup"
+    initdb -U $PGUSER -A md5 --pwfile=<(echo $PGPASSWORD) "$PGDATA"
+    OPT="unix_socket_directories"
+    sed -i "s|^#$OPT.*$|$OPT = '$PGDATA'|" "$PGDATA/postgresql.conf"
+    pg_ctl -D "$PGDATA" -l "$PGDATA/postgres.log"  start
+    createdb -U $PGUSER
+    psql -U $PGUSER -f "$PWD/db/init.sql"
+    pg_ctl -D "$PGDATA" stop
```

Comment:
I know I wrote this lol, but we probably wouldn't need to stop the server only to start it again if we had a variable keep track of whether the server started or not. honestly not a big deal though, this probably only happens once ever blue moon on project setup.

Reply:
alternatively, we can do a `postgres-ready` similar to `minio-ready` that does simple initialization if necessary